xdocs/index.xml

```diff
@@ -174,6 +175,14 @@ list of changes.
     <p>
       Please see the Apache Tomcat documentation for configuration specifics.
     </p>
+
+    <p>
+      When using unix domain sockets a cleanup is registered to delete the
```

Comment:
Last nit, use Unix Domain Sockets (uppercase) consistently.

Reply:
It's fixed.